.gitignore

```diff
@@ -37,6 +37,7 @@ oryx-build-commands.txt
 prof/
 tags
 TAGS
+*.swp
```

Comment:
A and don't forget to [revert](https://git-scm.com/docs/git-revert) or [drop](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) f190b2f